stable/pipelines/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: ">=1.8.1"
-version: 0.10.4
+version: 0.11.0
```

Comment:
lets move entire /pipelines/* changes to `/pipeloinesv2` chart to distinguish the work done here - it will give is better confidence in preserving backwards compatibility changes to older one - for already deployed chart fixes we would need to provide